components/datatypes/financial-account.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/datatypes/financial-account",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Financial Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:industry": ["financial_services"],
+  "description": "Financial account details, such as owners, balances, and types.",
+  "properties": {
+    "xdm:currentAccountBalance": {
+      "title": "Current Financial Account Balance",
+      "$ref": "https://ns.adobe.com/xdm/datatypes/currency",
+      "description": "Balance in the account."
+    },
+    "xdm:financialAccountID": {
+      "title": "Financial Account ID",
+      "type": "string",
+      "description": "Current account balance for a single user account."
+    },
+    "xdm:financialAccountName": {
+      "title": "Financial Account Name",
+      "type": "string",
+      "description": "Name assigned to the financial account."
+    },
+    "xdm:financialAccountOwner": {
+      "title": "Financial Account Owners",
+      "type": "array: mixin",

```

Comment:
This is not how we define array object. I think this is your VBA issue. Please fix this